src/components/activity/ActivityDirectiveForm.svelte

```diff
@@ -252,6 +256,13 @@
     effects.updateActivityDirective(plan_id, id, { name: initialValue }, user);
   }
 
+  function createPlaceholderTagObject(name: string): Tag {
+    function getColor() {
+      return hslToHex(360 * Math.random(), 25 + 70 * Math.random(), 82 + 10 * Math.random());
```

Comment:
Is this a random color generator? Can we put this guy in the color utils?

Reply:
Made the util but forgot to use it here 🤦